target/msm8916/oem_panel.c

```diff
@@ -138,6 +140,22 @@ static int init_panel_data(struct panel_struct *panelstruct,
 	int pan_type = PANEL_TYPE_DSI;
 
 	switch (panel_id) {
+	case CMI_NT35521_5P5_720P_VIDEO_PANEL:
```

Comment:
The changes here in target/msm8916/oem_panel.c are not needed. It is enough to add it in lk2nd/panel.